concent_api/concent_api/settings/base.py

```diff
@@ -223,6 +223,12 @@
             'level':     'DEBUG',
             'propagate': True,
         },
+        'concent.crash': {
```

Comment:
Do we need a separate logger here? `'request.django'` is configured in the same way

Reply:
@dybi According to description, yes

> Create a logger called concent.crash. It should have the same handlers as django.request so that things logged there get reported to Sentry.

But personally I think we might use 'request.django' as well.

Reply:
Ok, missed that. It such case - just merge it ;)

Reply:
We don't strictly need it but I think using `django.request` everywhere would be pretty misleading and might lead to people using it the wrong way. The hierarchy of loggers mirrors the code structure - as  the typical usage where the dotted path of the module becomes the name of hte logger indicates. The fact that two loggers have the same handlers does not mean that they have to be merged into a single logger.

I think that we should eventually replace all other uses of `django.request` with the new logger and leave the original logger to Django.